subsys/bluetooth/shell/iso.c

```diff
@@ -237,6 +422,18 @@ SHELL_STATIC_SUBCMD_SET_CREATE(iso_cmds,
 	SHELL_CMD_ARG(send, NULL, "Send to ISO Channel", cmd_send, 1, 0),
 	SHELL_CMD_ARG(disconnect, NULL, "Disconnect ISO Channel",
 		      cmd_disconnect, 1, 0),
+#if defined(CONFIG_BT_ISO_BROADCAST)
+	SHELL_CMD_ARG(big-create, NULL,
```

Comment:
I usually avoid using the same prefix for multiple commands to make tab completion more efficient, either move to be a suffix e.g. create-big or add a subcmd big.

Reply:
Sure, we can do that.